samples/cellular/modem_shell/src/sock/sock_shell.c

```diff
@@ -165,13 +167,79 @@ static const char sock_rai_usage_str[] =
 	"                            option before the next send call.\n"
 	"  -h, --help,               Shows this help information";
 
+static const char sock_setopt_usage_str[] =
+	"Usage: sock option set -i <socket id> -o <socket option> -v <value>\n"
+	"       [--level <level>]\n"
```

Comment:
@stig-bjorlykke, let me know your thoughts about this and whether you think this would be enough, or do we still need something like PR #19818?

Reply:
This PR has gone into `Stale`. I will proceed with this idea at some point and try to get this change into main before next NCS release.